src/components/pageBody/proposal/governanceProposals/proposalList/proposalInfo/ProposalInfo.tsx

```diff
@@ -24,8 +24,8 @@ function ProposalInfo({ proposalContract, proposalIndex }: any) {
     objectConverted.length !== 0 && Object.keys(proposalInfo.detail)[0];
   return (
     <Flex direction="column" gap={4}>
-      <>{console.log(proposalInfo)}</>
-      <>{console.log(proposalType)}</>
+      {/* <>{console.log(proposalInfo)}</>
```

Comment:
hmm Comments in code?  I think you mentioned you're coming back to fix these, yeah?